clang_tools/wheel_install.py

```
@@ -0,0 +1,182 @@
+        f"Latest stable version: {latest}\n"
+        f"Available versions (sample): {sample}\n"
+        f"Run `pip index versions {tool}` to see all available versions.",
+    )
+
+
+def _is_version_installed(tool: str, version: str) -> Optional[Path]:
+    """Return the tool path if the installed version matches, otherwise None."""
+    existing = shutil.which(tool)
+    if not existing:
+        return None
+    result = subprocess.run([existing, "--version"], capture_output=True, text=True)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)